packages/react-app/src/components/main-view.tsx

```diff
@@ -114,7 +125,9 @@ function MainView({ children, toggleTheme }: Props) {
           </Header>
         </HeaderWrapperStyled>
 
-        {page === ENUM_PAGES.List ? (
+        {page === undefined && <Skeleton />}
```

Comment:
I think skeleton is not the best think here we might want a spinner and the content should take the whole height
![image](https://user-images.githubusercontent.com/13890618/162463087-c82f48c7-10c0-4d9a-b67a-4315b6df58d9.png)

Reply:
Can be done later tho